### PR TITLE
Update licensing terms

### PR DESCRIPTION
**Pull request by nkramer**

### CODE REVIEW

File: concepts/teams-licenses.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: "Licenses for subscribing to chat messages"
+description: "Licenses for subscribing to chat messages"
+author: "nkramer"
+localization_priority: Priority
+ms.prod: "microsoft-teams"
+---
+
+# Licenses for subscribing to chat messages
```
Comment (Lauragra) [inline concepts/teams-licenses.md:9]: How are we planning to surface this article? In the concept TOC, under Teamwork? https://docs.microsoft.com/en-us/graph/teams-concept-overview

File: concepts/teams-licenses.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Creating chat message subscriptions with application permissions for `/teams/allMessages` and `/chats/allMessages` will soon require one of the following licenses:
+
```
Comment (Lauragra) [inline concepts/teams-licenses.md:12]: We're saying "will soon" here, but in the previous topic we say "Starting in August 2020". Does that mean that the licenses are required now? If so, we should change "will soon" to "requires".

Comment (nkramer) [reply concepts/teams-licenses.md:12]: No, it's not checked in yet -- will happen soon, though.

Comment (nkramer) [reply concepts/teams-licenses.md:12]: I've had a TOC link, but the main path to get to it is from the create subscription page.

File: concepts/teams-licenses.md (hunk 1 above)
Comment (nkramer) [reply concepts/teams-licenses.md:9]: I've added a TOC link, but the main path to get to it is from the create subscription page.

**Pull request merged**